### ISSUE TITLE
Battle standards not affecting teammates from the opposite faction during cross-faction battlegrounds.

### ISSUE THREAD

**Issue by waowplayer** [chromiecraft/chromiecraft#384]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x ] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [x ] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39

##### CURRENT BEHAVIOUR:

When you use a battle standard during a cross-faction battleground, it will not buff teammates from the opposite faction.

In the example screenshot below, you can see the Alliance player Blobbo benefitting from the Alliance Battle Standard buff but not his Horde teammate Illya :
https://i.gyazo.com/e77114d677152142e5f41e239b54efcb.jpg

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Battle standards should buff all of your teammates when you use them.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Join a cross-faction battleground (e.g, Warsong Gulch)
2. Use a battle standard next to your teammates
3. Notice how your teammates from the opposite faction are not getting the buff

##### EXTRA NOTES:

Items ID:

https://wowgaming.altervista.org/aowow/?item=18606

https://wowgaming.altervista.org/aowow/?item=18607

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/ee1918a3e9bc355fbd6f8d58d77760fb61fdbff7

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Helias** [comment]
this is a problem of https://github.com/azerothcore/mod-cfbg/

**Comment by Stifler82** [comment]
But this is not a bug, it is due to the fact that the two factions "Theoretically" should not be mixed in the group...

**Comment by Helias** [comment]
bug or not we need to make it work with the CFBG :pensive:

**Comment by Stifler82** [comment]
I think we now have enough players to restore everything to normal, and then the bug resolves itself..

**Comment by FrancescoBorzi** [comment]
https://github.com/azerothcore/mod-cfbg/issues/41

**Comment by FrancescoBorzi** [comment]
> @Helias
> I think we now have enough players to restore everything to normal, and then the bug resolves itself..

I don't think we are there yet, but regardless of that, the bug in the CFBG is a valid one
